cogs/mod.py

```diff
@@ -1037,7 +1043,13 @@ def check(m):
     async def whitelist(self, ctx):
         """Users who will be able to use the bot"""
         if ctx.invoked_subcommand is None:
+            name = list(set([x.name for x in self.bot.get_all_members() if x.id in self.whitelist_list]))
             await send_cmd_help(ctx)
+            if name:
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
+            await self.bot.say(box('Whitelisted users:\n' + userlist))
```

Comment:
Unsure if what to display ontop.
My suggestions are

Whitelisted users :
Whitelist :